lib/process-manager.ts

```diff
@@ -370,6 +371,15 @@ export abstract class ProcessManager {
 
 		this.listen();
 	}
+	static exec(args: string | string[], execOptions?: child_process.ExecOptions | child_process.ExecFileOptions) {
+		if (Array.isArray(args)) {
+			const cmd = args.shift();
+			if (!cmd) throw new Error(`You must pass a command to ProcessManager.exec.`);
+			return util.promisify(child_process.execFile)(cmd, args, execOptions);
```

Comment:
I would manually promisify instead of using `util`.

Reply:
Alright.